resources/english.yml

```diff
@@ -360,6 +360,68 @@ msg_err_enemy_war_not_on_edge_of_town: '&6[War]&c Only allowed to attack the bor
 msg_err_enemy_war_cell_already_under_attack: '&6[War]&c This area is already under attack by %s.'
 msg_err_enemy_war_reached_max_active_flags: '&6[War]&c You can''t attack more than %d areas at once.'
 
+#Siege war happy-path messages
```

Comment:
All new strings need to go at the end of the file, for the translators of the other language files.